COMPILING.md

```diff
@@ -0,0 +1,141 @@
+## Points to note
+
+- Jamulus can be compiled for Linux, Windows and macOS. However, the preferred method of supporting these platforms is to use the binaries generated by the autobuild process in the Jamulus repository.
+- For unattended installs, see the contributed [installation scripts](https://github.com/jamulussoftware/installscripts)
+- There are reports from people who successfully compile and run Jamulus on BSDs.
+- Android and iOS are not officially supported, therefore there are no instructions yet.
```

Comment:
iOS could be documented since it's not that difficult. @ngocdh

Reply:
I don't know how to suggest a change here. Could you help? This is the text I prepared (credit @jeroenvv):

 - Make sure to install qt5 with the qt installer (not homebrew), and explicitly select iOS when choosing the Qt version
 - Run /path/to/qt/5.15.2/ios/bin/qmake -spec macx-xcode Jamulus.pro
 - Open the generated .xcodeproject in XCode, check the Signing & Capabilities and configure a team 
 - To run on a iOS device, build and run on the device, you'll have first trust your developer profile in device's Settings.

Reply:
Thanks - that's now in.